checks/cs50/2017/x/mario/mario.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Mario(Checks):
+    @check()
+    def exists(self):
+        """mario.c exists."""
+        super(Mario, self).exists("mario.c")
```

Comment:
We didn't decide on what the best route was so I did it this way to resemble the other checks. Personally, I think `Checks.exists` is just as clear as `super().exists` when we have single inheritance (that's what StackOverflow seems to suggest anyway).